### ISSUE TITLE
[BUG] fe-1 does not properly disconnect from a LAO

### ISSUE THREAD

**Issue by Tyratox** [dedis/popstellar#1132]
### Description (Actual behavior)
When disconnecting from a LAO, the user is navigated to the correct screen but  fe-1 does not unset `currentId` in the lao reducer.

### Expected behavior
The `currentId` in the lao reducer should be set to undefined after disconnecting.

### How to reproduce
1. Connect to a LAO
2. Disconnect from a LAO
3. Check the redux state of the lao reducer

### Version & Environment
This bug was reproduced on:
- Commit: 3cd501d7038af61d4d197607970869b9308a8cf4

##### Front-ends:
- [x] Fe1-Web (please include browser's names & version)
- [ ] Fe2-Android (please specify if phone or emulation, and Android version)
- [ ] Not applicable

##### Back-ends:
- [ ] Be1-Go
- [ ] Be2-Scala
- [x] Not applicable

### Workaround
Connect to a different lao

### Impact
Certain screens such as the wallet will behave differently (as if you were still connected to the previous LAO)

### Possible root cause
The issue is not easy to fix because of how react-navigation loads and unloads screens. If the value is immediately unset, the application crashes because the lao screen expects it to be set but is not instantly unloaded.
A workaround would be to delay this update but it is not clear how long: A long delay will make the application crash if you quickly re-connect to a second lao (because you are then "disconnected" while the lao navigation is open) and a short delay might not work on all devices. I currently do not see a nice way to fix it. Maybe react error boundaries would help catching and handling the exceptions correctly.

**Comment by pierluca** [comment]
If I understand correctly the root cause analysis above, the issue is that you want to navigate before clearing the state, because clearing the state first and navigating afterwards might fail, due to the way react navigation works.
You don't want to use delays, because you (correctly) analyze that they might introduce their own set of (subtle) bugs.
Since we can't use delays, we need to do so in reaction to some event, and we know that changing page must happen before the unsetting, as the alternative leads to a crash.

Therefore, how about unsetting the current Lao ID when loading the correct page, triggering this logic by passing an extra argument to the page via react navigation ?  
I.e. we tell the application to change page on disconnect, passing an extra parameter, and as the page loads, it checks the parameter and clears the state.
What do you think? Is it feasible?

**Comment by Tyratox** [comment]
Yes this would be feasible and I tried it here https://github.com/dedis/popstellar/commit/15df578b4c1a4ba412e544ced4baa1afda76f13e.

Unfortunately it still crashes because apparently react-navigation still keeps the past screen loaded for some short time..
